src/test/resources/net/syscon/elite/api/resource/impl/offender_detail_aliases.json

```diff
@@ -131,7 +130,7 @@
     }
   ],
   "status": "ACTIVE IN",
-  "legalStatus": "Convicted",
+  "legalStatus": "SENTENCED",
```

Comment:
So to confirm, this only affects offender search?

Reply:
Yes